### PR TITLE
feat: Add support of .aab packages

### PR DESCRIPTION
**Pull request by mykola-mokhnach**
The idea is simple. If an .aab package is provided we transform it to a universal .apk using bundletool and cache this .apk, because it is too expensive to do such action for each test if the same app is being tested (which is usually the case)

### CODE REVIEW

**Review by KazuCocoa — APPROVED** [review]
seems good, overall.
Should we append description in `appium:app` in README?
So, users have two options for `.aab`. One is by them, and the other is appium converts it to universal apk by this pr.

**Comment by mykola-mokhnach** [comment]
Updated readme

**Review by KazuCocoa — APPROVED** [review]

File: lib/driver.js [hunk 1]
```diff
@@ -219,6 +233,46 @@ class EspressoDriver extends BaseDriver {
     }
   }
 
+  async onPostConfigureApp ({cachedAppInfo, isUrl, appPath}) {
```
Comment (KazuCocoa) [inline lib/driver.js:236]: noticed still this `appPath` could have `.zip` extension, so `.zip` file always handles as `this.adb.extractUniversalApk` even it is `apk` after unzip.
https://github.com/appium/ruby_lib_core/issues/370

Comment (KazuCocoa) [reply lib/driver.js:236]: https://github.com/appium/appium/pull/16548

**Pull request merged**